test/library/standard/Path/victor-ludorum/cygwin/testcases_isAbsPath.future

```diff
@@ -1,3 +1,11 @@
+   // Test file for isAbsPath() function for cygwin
```

Comment:
Sorry, the .future file is needed *in addition* to the .chpl file.  Was this just misreading the instructions, or is there a particular part of the description of futures that led you to this conclusion?

Reply:
In the doc which you have suggested , only 3 files are mentioned .future, .good and .bad . Therefore I have added this only.

Reply:
"To mark a test as a future, you add a .future file sharing the same base name as the test. For example, adding a hi.future file would make the simple correctness test at the start of this document into a future test." (second paragraph of the futures section)

This snippet indicates to me that the .future file should be additional, not replacing the .chpl file.

Reply:
Okay!! I think I have not understood it correctly . Thanks !!

Reply:
I have added testcases_isAbsPath.chpl file .